Renegade X Launcher/Views/ApplyUpdateWindow.xaml.cs

```diff
@@ -337,8 +339,9 @@ public void Close_Click(object sender, RoutedEventArgs e)
 
         public void Cancel_Click(object sender, RoutedEventArgs e)
         {
-            CancellationTokenSource.Cancel();
+            _cancellationTokenSource.Cancel();
             this.StatusMessage = "Operation cancelled by User";
+            Application.Current.Shutdown();

```

Comment:
`Cancel` should not close the entire application; it should only cancel the patch